docs/hooks/glossary.py

```diff
@@ -0,0 +1,80 @@
+import os
+import re
+import tomllib
+from typing import Any
+
+def load_glossary(file_path="docs/glossary.toml"):
+  if not os.path.exists(file_path):
+    return {}
```

Comment:
this should error, not fail silently. for this PR, you can just include an empty one or port over the existing glossry

Reply:
Will add an empty. I made a separate PR for the glossary term file #34232

Reply:
pushed